android/app/build.gradle

```diff
@@ -9,6 +9,10 @@ apply plugin: 'kotlin-kapt'
 apply plugin: 'androidx.navigation.safeargs'
 
 android {
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8

```

Comment:
Is our app still compatible with android 4.4?

Reply:
I've added these lines because after adding Usb Serial library I got a compilation error

Then I've searched the web for my error and the [first suggestion on StackOverflow](https://stackoverflow.com/questions/49891730/invoke-customs-are-only-supported-starting-with-android-0-min-api-26) was to add these.

Reply:
I suppose our app is still compatible with Android 4.4, why not?